charts/kueue/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: kueue
+description: A Helm chart for Kubernetes
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.3.0
```

Comment:
As seen in https://helm.sh/docs/topics/charts/#the-chartyaml-file , `version` refers to the revision of the chart it self, and `appVersion` is for the managed app which in this case is `0.3.0` So in this case `version` should be `0.1.0`

Reply:
Thanks, fixed it.